api/taskgroup.go

```diff
@@ -187,6 +186,7 @@ func (h TaskGroupHandler) Update(ctx *gin.Context) {
 			})
 		return
 	}
+	db = db.Omit("BucketID")
```

Comment:
hint: Just adding the BucketID here. Not using Omit(clause.Association) here because during the submit flow, we rely on the ORM to create the []Tasks.